### PR TITLE
⏮️ Preprocess for quant as true for bert model

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes

Enable quantization preprocess by default for bert model, as we know that would avoid some bad samples from olive optimization for bert cases. 

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by jambayk — APPROVED** [review]
Thanks! Can you update the PR description before merging with the reason for this so we have a record of it in the commit history?

**Comment by trajepl** [comment]
> Thanks! Can you update the PR description before merging with the reason for this so we have a record of it in the commit history?

done! thanks.

File: .azure_pipelines/performance_check/configs/bert.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
         "quantization": {
             "type": "OnnxQuantization",
             "config": {
+                "quant_preprocess": true,
```
Comment (guotuofeng) [inline .azure_pipelines/performance_check/configs/bert.json:61]: could we add this knowledge when generating auto-tune config?

Comment (trajepl) [reply .azure_pipelines/performance_check/configs/bert.json:61]: Yes~ We should. It is a good experience to run pre process for quantization.

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**